jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -97,6 +97,7 @@ class GemUtils {
                 // should already have taken care of, see #79
                 environment 'JBUNDLE_SKIP', true
                 environment 'JARS_SKIP', true
+                environment 'GEM_HOME', destDir.absolutePath
```

Comment:
I see what you mean. I think it makes more sense to directly set `GEM_PATH` to `destDir.absolutePath`. Something similar to `[destDir.absolutePath, environment['GEM_PATH']].join(File.pathSeparator)` could be used, but that might break the isolation and create conflicts with other ruby installation.